l3kernel/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ this project uses date-based 'snapshot' version identifiers.
 
 ## [Unreleased]
 
+### Added
+- `dflt` alias to str_convert
```

Comment:
Is there any real reason for saving the vocals in that name? I think it is rather strange looking (assuming it stands for "default"

Reply:
no, default is fine too, I will change to it. I only started with variants of utf8xx when struggling to find a name and so was in "abbreviation mode" and then the open type `dflt` jumped into my mind.